public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -387,6 +388,9 @@ export class UnthemedDashboardPage extends PureComponent<Props, State> {
               <SubMenu dashboard={dashboard} annotations={dashboard.annotations.list} links={dashboard.links} />
             </section>
           )}
+          {config.featureToggles.angularDeprecationUI && dashboard.hasAngularPlugins() && dashboard.uid !== null && (
```

Comment:
Maybe we should have a check here that this is only shown to dashboard users who have edit permissions?

Reply:
we could but then it makes it difficult for a viewer to discover that a dashboard they rely on has this issue